.github/workflows/build_and_test.yml

```diff
@@ -75,33 +81,34 @@ jobs:
       - name: Build OSX Client
         run: npx gulp build --platform=osx
 
-  # TODO: setup fastlane for iOS
-  
-  # ios:
-  #   name: iOS
-  #   runs-on: macos-11
-  #   steps:
-  #     - name: Checkout
-  #       uses: actions/checkout@v2.3.4
+  ios:
+    name: iOS
+    runs-on: macos-11
+    timeout-minutes: 10
+    environment: Test
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
 
-  #     - name: Install Node
-  #       uses: actions/setup-node@v2.2.0
-  #       with:
-  #         node-version: 16
-  #         cache: npm
+      - name: Install Node
+        uses: actions/setup-node@v2.2.0
+        with:
+          node-version: 16
+          cache: npm
 
-  #     - name: Install NPM Dependencies
-  #       run: npm ci
+      - name: Install NPM Dependencies
+        run: npm ci
 
-  #     - name: Set XCode Version
-  #       run: sudo xcode-select -switch /Applications/Xcode_12.5.app
+      - name: Set XCode Version
+        run: sudo xcode-select -switch /Applications/Xcode_12.5.app
 
-  #     - name: Build iOS Client
-  #       run: npm run action apple/scripts/ios_package_remote
+      - name: Build iOS Client
+        run: npm run action apple/scripts/ios_package_remote
```

Comment:
It seems this is failing.

Perhaps echo an innocuous, non-secret variable to validate the environment?
(don't echo the secrets)

Reply:
I think it's just because I removed the `env` since we thought `environment` passed them in explicitly

Reply:
Oh and btw github is smart enough to censor secrets if you happen to echo them in a job, even after you base64 it